GPU/Common/FramebufferManagerCommon.cpp

```diff
@@ -121,6 +121,7 @@ VirtualFramebuffer *FramebufferManagerCommon::GetVFBAt(u32 addr) const {
 		VirtualFramebuffer *v = vfbs_[i];
 		if (v->fb_address == addr) {
 			// Could check w too but whatever
+			// NOTE: This gets the OLDEST image at the address - is that good?
```

Comment:
Hmm, `v` is probably more appropriately `candidate` and `match` maybe is better `result`.  It replaces `result` with `candidate` if `result->last_frame_render < candidate->last_frame_render`.  That sounds like it takes the latest.

But the names aren't terribly helpful, I suppose.

-[Unknown]

Reply:
Ah my mind wrongly assumed that v was the iterated one, since I usually make those variables short. Duh.